couchable/core.py

```diff
@@ -519,10 +542,10 @@ def store(self, what, skip=None, additiveOnly=False):
         #log_internal.error("outside for")
         #print "outside for", self._obj_by_id.items(), store_list
 
-        del self._done_dict
-        del self._cycle_set
-        del self._skip_list
-        del self._additiveOnly
+        self._done_dict = collections.OrderedDict()
+        self._cycle_set = set()
+        self._additiveOnly = None
+        self._skip_list = []
```

Comment:
Better to reset attributes than adding/removing them on runtime, see above for initialization comment